index.html

```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
+        <h2 class="ff-crete mg-8 center">My Recent Works</h2>
+        <div class="featured">
+            <img src="assets/img/img-placeholder.png" alt="">
+            <h3 class="mg-1 ff-crete" >Multi-Post Stories</h3>
+            <p class="ff-inter"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+            <ul class="tag tag-feature mg-1 ff-inter flex">
+                <li > css </li>
+                <li > html </li>
+                <li> bootstrap </li>
+                <li> Ruby </li>
+            </ul>
+            <button class="btn" type="button"> See Project </button>
+        </div>
+        <div class="flex project-container">
```

Comment:
- [ ] The main container of the projects should have  a `display:grid`

Reply:
![Screenshot from 2022-04-23 19-38-13](https://user-images.githubusercontent.com/24270269/164937724-87444264-f405-46a3-aba6-ba4fe6599f5c.png)

@GzimAsani Thank you for the review. I am of the mindset that the display: grid container is for both the feature project and other projects.